### DISCUSSION TITLE
Deploy en Azure. Error de jar

### DISCUSSION THREAD

**Discussion by tobiascastaneda2000** [dds-jv/foro#406] [Consultas de Arquitectura]
Buenas noches! Queriamos informar que estabamos haciendo el deploy utilizando Azure, y al mommento del despliegue nos aparece el mismo error: 

`Deployment Failed, Error: More than one package matched with specified pattern: *.jar. Please restrain the search pattern.`

Por lo que averiguamos hay un error parecido que tiene que ver con el archivo zip, en el que basicamente teniamos que especificar el archivo zip especifico que queriamos utlizar. 

Sin embargo, nosotros usamos un archivo jar, y no sabemos por que nos surge este error. ¿Es por que hay que especificar el archivo  jar? ¿Como lo hacemos a traves de Azure?¿Es otro error?

Aclaracion: Para poder hacer el deploy tuvimos que crear un nuevo repositorio PUBLICO y pasarle todos los datos del repositorio privado, por que de lo contrario no tomaba como opccion para poder hacer el despliegue al momento de autenticar el github. NO nos aparecia el repo privado de nuestro usuario pero si aparecia los publicos

Aclaracion2: Para hacer el deploy por azure nos estamos guiando de este video  a rajatabla: https://www.youtube.com/watch?v=idU176MXgOU&t=705s

Disculpe las molestias. Espero su respuesta

**Comment by flbulgarelli** [comment]
Hola @tobiascastaneda2000,

Me falta contexto para poder entender tu consulta, pero a priori no publiquen por favor su código. El mismo debe estar siempre privado.

Por otro lado, pasen el enlace al repositorio (privado) para que podamos ayudarles. Además, expliquen (por escrito) qué pasos siguieron y qué configuraciones generaron. 

Saludos

**Reply by tobiascastaneda2000** [reply]
Buenas noches!!

Disculpe la demora. Al final decidimos utilizar Heroku para deplegar la app y por suerte funciono. En esta faltaba que genere el .jam y especificar que archivo tiene el main.

Muchas gracias! Disculpe las molestias
